DisguiseUnityRenderStream/DisguiseRenderStream.cs

```diff
@@ -363,18 +365,23 @@ static void ProcessFrameData(in FrameData receivedFrameData)
                     ++i;
                 }
             }
+
+            parameters.Dispose();
+            imageData.Dispose();
         }
     
         public static IEnumerator AwaitFrame()
         {
             if (awaiting)
                 yield break;
             awaiting = true;
+            
+            var waitForEndOfFrame = new WaitForEndOfFrame();
```

Comment:
It's not used outside of this method so it makes sense to keep it inside. There's a catchy name for this principle, but I can't remember it.

Reply:
I know, it was more to avoid a heap allocation every frame.  But if you think there isn't too much of it then it is indeed better to keep it as a local variable.

Reply:
I don't think it's allocating every frame. The `new` is not inside the loop.